lib/models/Upload.ts

```diff
@@ -3,13 +3,15 @@ type TUpload = {
   size?: number
   offset: number
   metadata?: string
+  creation_date?: string
 }
 
 export default class Upload {
   id: TUpload['id']
   metadata?: TUpload['metadata']
   size?: TUpload['size']
   offset: TUpload['offset']
+  creation_date?: TUpload['creation_date']
```

Comment:
It's never undefined
```suggestion
  creation_date: TUpload['creation_date']
```